helm/chart/templates/hpa-controller-keptn-metrics-rbac.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ include "chart.fullname" . }}-hpa-controller-keptn-metrics
+  name: hpa-controller-keptn-metrics
   namespace: {{ .Release.Namespace | quote }}
   labels:
   {{- include "chart.labels" . | nindent 4 }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: '{{ include "chart.fullname" . }}-server-resources'
+  name: 'server-resources'
```

Comment:
was this intentional?

Reply:
yes this is removed everywherelse, so the chart was wrong..